wp-content/plugins/AB-Content-Alterer/ab-content-alterer.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/*
+Plugin Name:  AB Content Alterer
+Description:  WordPress Content Alterer: Change Unbounce and Leadpages in content to <strong>Instapage</strong>
+Version:      0.7
+Author:       andrzej.bialowas@instapage.com
+*/
+defined( 'ABSPATH' ) or die( 'No script kiddies please!' );
+add_action('the_content', 'abAlterContent');
+function abAlterContent($abTextToAlter) {

```

Comment:
DocBlock would be helpful to describe what this function does

Reply:
Also - to avoid function name conflicts and keep the function names clear, create a class for your plugin.
https://code.tutsplus.com/articles/create-wordpress-plugins-with-oop-techniques--net-20153